webapp/channels/src/packages/mattermost-redux/src/actions/channels.ts

```diff
@@ -45,6 +46,9 @@ import {getMissingProfilesByIds} from './users';
 
 import {General, Preferences} from '../constants';
 
+const missingChannelWait = 100;
+const maxMissingChannelsBatchSize = 1; // max batch size is 1 because we don't have an API to get multiple channels by IDs
```

Comment:
Nit, these could actually live in `fetchMissingChannels` since they're not used elsewhere. The only reason that the other ones are defined as constants outside of the components is because they're reused in multiple places